### ISSUE TITLE
Example API usage specifies config for container_init, but takes container argument

### ISSUE THREAD

**Issue by mthaddon** [lxc/pylxd#57]
examples/api_test.py has the following:

```
config = {'name': CONTAINER_NAME,
          'source': {'type': 'none'}}
lxd.container_init(config)

```
But pylxd/container.py has:

```
def container_init(self, container):
        return self.connection.get_object('POST', '/1.0/containers',
                                          json.dumps(container))
```

It looks like you're passing a config, not a container, and it's not documented what the options to that config are.

**Comment by mthaddon** [comment]
Seems like the options come from https://github.com/lxc/lxd/blob/master/specs/rest-api.md

**Comment by rockstar** [comment]
Fixed in #86
